src/tool-header.scss

```diff
@@ -0,0 +1,295 @@
+  ),
+  'critical':  (
+    'iconColor': var(--fdTool_Header_Icon_Color_Critical),
+    'iconColorInverted': var(--fdTool_Header_Icon_Color_Inverted_Critical),
+    'bgColorInverted': var(--fdTool_Header_Icon_Background_Color_Inverted_Critical)
+  ),
+  'negative':  (
+    'iconColor': var(--fdTool_Header_Icon_Color_Negative),
+    'iconColorInverted': var(--fdTool_Header_Icon_Color_Inverted_Negative),
+    'bgColorInverted': var(--fdTool_Header_Icon_Background_Color_Inverted_Negative)
+  ),
+  'neutral':  (
+    'iconColor': var(--fdTool_Header_Icon_Color_Neutral),
+    'iconColorInverted': var(--fdTool_Header_Icon_Color_Inverted_Neutral),
+    'bgColorInverted': var(--fdTool_Header_Icon_Background_Color_Inverted_Neutral)
+  ),
+);
+
+$fd-tool-header-object-status-semantic-values: (
+  'positive':  (
+    'color': var(--fdTool_Header_Object_Status_Color_Positive),
+    'iconColorInverted': var(--fdTool_Header_Object_Status_Text_Color_Inverted),
+    'bgColorInverted': var(--fdTool_Header_Object_Status_Background_Color_Inverted_Positive)
+  ),
+  'critical':  (
+    'color': var(--fdTool_Header_Object_Status_Color_Critical),
+    'iconColorInverted': var(--fdTool_Header_Object_Status_Text_Color_Inverted),
+    'bgColorInverted': var(--fdTool_Header_Object_Status_Background_Color_Inverted_Critical)
+  ),
+  'negative':  (
+    'color': var(--fdTool_Header_Object_Status_Color_Negative),
+    'iconColorInverted': var(--fdTool_Header_Object_Status_Text_Color_Inverted),
+    'bgColorInverted': var(--fdTool_Header_Object_Status_Background_Color_Inverted_Negative)
+  ),
+  'neutral':  (
+    'color': var(--fdTool_Header_Object_Status_Color_Neutral),
+    'iconColorInverted': var(--fdTool_Header_Object_Status_Text_Color_Inverted),
+    'bgColorInverted': var(--fdTool_Header_Object_Status_Background_Color_Inverted_Neutral)
+  ),
+  'informative':  (
+    'color': var(--fdTool_Header_Object_Status_Color_Informative),
+    'iconColorInverted': var(--fdTool_Header_Object_Status_Text_Color_Inverted),
+    'bgColorInverted': var(--fdTool_Header_Object_Status_Background_Color_Inverted_Informative)
+  )
+);
+
+.#{$block} {
+  @include fd-reset();
+
+  @include fd-flex-vertical-center() {
+    justify-content: space-between;
+  }
+
+  width: 100%;
+  padding: 0 0.5rem 0 1rem;
+  height: $fd-tool-header-height;
+  background: var(--sapShellColor);
+  box-shadow: inset 0 -0.0625rem 0 0 var(--sapShell_BorderColor);
+
+  // ELEMENTS
+  &__group {
+    @include fd-reset();
+    @include fd-flex-vertical-center();
+  }
+
+  &__element {
+    @include fd-reset();
+
+    @include fd-flex-center() {
+      flex-shrink: 0;
+    }
+
+    max-width: 100%;
+    overflow: hidden;
+    height: $fd-tool-header-height;
+
+    &:not(:last-child) {
+      @include fd-set-margin-right(0.5rem);
+    }
+  }
+
+  &__logo {
+    @include fd-reset();
+
+    width: auto;
+    height: 1.625rem;
+    object-fit: cover;
+    background-size: contain;
```

Comment:
Maybe add a modifier for 'cover' option?

Reply:
it's actually not needed, removed it.